lib/HttpClient/CurlClient.php

```diff
@@ -251,19 +251,19 @@ public function request($method, $absUrl, $headers, $params, $hasFile)
         \array_push($headers, 'Expect: ');
 
         $absUrl = Util\Util::utf8($absUrl);
-        $opts[CURLOPT_URL] = $absUrl;
-        $opts[CURLOPT_RETURNTRANSFER] = true;
-        $opts[CURLOPT_CONNECTTIMEOUT] = $this->connectTimeout;
-        $opts[CURLOPT_TIMEOUT] = $this->timeout;
-        $opts[CURLOPT_HTTPHEADER] = $headers;
-        $opts[CURLOPT_CAINFO] = Stripe::getCABundlePath();
+        $opts[\CURLOPT_URL] = $absUrl;
+        $opts[\CURLOPT_RETURNTRANSFER] = true;
+        $opts[\CURLOPT_CONNECTTIMEOUT] = $this->connectTimeout;
+        $opts[\CURLOPT_TIMEOUT] = $this->timeout;
+        $opts[\CURLOPT_HTTPHEADER] = $headers;
+        $opts[\CURLOPT_CAINFO] = Stripe::getCABundlePath();
         if (!Stripe::getVerifySslCerts()) {
-            $opts[CURLOPT_SSL_VERIFYPEER] = false;
+            $opts[\CURLOPT_SSL_VERIFYPEER] = false;
         }
 
-        if (!isset($opts[CURLOPT_HTTP_VERSION]) && $this->getEnableHttp2()) {
+        if (!isset($opts[\CURLOPT_HTTP_VERSION]) && $this->getEnableHttp2()) {
             // For HTTPS requests, enable HTTP/2, if supported
-            $opts[CURLOPT_HTTP_VERSION] = CURL_HTTP_VERSION_2TLS;
+            $opts[\CURLOPT_HTTP_VERSION] = \CURL_HTTP_VERSION_2TLS;
```

Comment:
Check that this still works if the constant is manually defined via `define()`

Reply:
It does, `define` ignores the current namespace and defines the new variable into the root namespace.